airflow/cli/cli_config.py

```diff
@@ -818,6 +823,13 @@ def string_lower_type(val):
     action="store_true",
 )
 
+# IMPORTANT NOTE!
+#
+# Celery configs below have explicit fallback values because celery provider defaults are not yet loaded
+# via provider at the time we parse the command line, so in case it is not set, we need to have manual
+# fallback
+# DO NOT REMOVE THE FALLBACKS even if you are tempted to.
+# TODO: possibly move the commands to providers but that could be big performance hit on the CLI
```

Comment:
Should we rather have it at the top of the page or elsewhere?

Reply:
No - because it only applies to celery configuration.